src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -4548,5 +4554,31 @@ public unsafe bool IsReplicationCertificateByPublicKeyPinningHash(TransactionOpe
 
             return false;
         }
+
+        private void AssertLicenseLimitsAddDatabase(ServerStore serverStore, DatabaseRecord databaseRecord, ClusterOperationContext context)
```

Comment:
why this isn't a part of `AssertLicenseLimits` of the `AddDatabaseCommand`?

Reply:
Done